### ISSUE TITLE
MongoError: cannot use the part (roles of roles.__global_roles__) to traverse the element 

### ISSUE THREAD

**Issue by jimmiebtlr** [alanning/meteor-roles#80]
The user in question has roles that are not part of a group.  I am trying to add a role which is part of 

    Roles.GLOBAL_GROUP

and I get the following error 

    MongoError: cannot use the part (roles of roles.__global_roles__) to traverse the element 

Any insight into it's meaning would be great, thanks.  Not really sure where to start on this.

**Comment by alanning** [comment]
I would guess that it has to do with the structure of the `roles` field on the user object.

The choice to use groups should be a global one; either all users should use groups or none of your users should use them.  To convert from no-groups to groups, you would manually change the structure to match what the roles package expects as shown here:  https://github.com/alanning/meteor-roles#changes-to-default-meteor-behavior

Here's a small script that I used when we did the conversion for our startup:  https://gist.github.com/alanning/0686f29c71b8cab54cc7

Any issues, let me know.
